content/docs/rendering-elements.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: rendering-elements
-title: تصيير العناصر
+title: تصدير العناصر
```

Comment:
Thank you @E3saR33  for your contribution.
Look to the [Glossary](../../wiki/Glossary)